skfem/mesh/mesh.py

```diff
@@ -418,6 +418,26 @@ def __post_init__(self):
                                "to C_CONTIGUOUS.")
             self.t = np.ascontiguousarray(self.t)
 
+        # add boundary tags for the default cubes
+        if ((self._boundaries is None
+             and self.doflocs.shape[1] == 2 ** self.dim())):
+            boundaries = {}
+            # default boundary names along the dimensions
+            minnames = ['left', 'bottom', 'front']
+            maxnames = ['right', 'top', 'back']
+            for d in range(self.doflocs.shape[0]):
+                dmin = np.min(self.doflocs[d])
+                ix = self.facets_satisfying(lambda x: np.isclose(x[d], dmin))
```

Comment:
I think `==` is more correct than `isclose` here (we know that `dmin` is taken on at least once, and is the face really nonplanar? I mean don't all points of that face have identical coordinates in the normal direction?) but i'm sure you've already considered that and decided the reverse.